xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/docextra.vm

```diff
@@ -47,6 +47,7 @@
     $xwiki.ssfx.use('uicomponents/widgets/upload.css', true)
     $xwiki.jsfx.use('js/xwiki/viewers/attachments.js', {'forceSkinAction': true, 'language': ${xcontext.locale}})
     $xwiki.ssfx.use('js/xwiki/viewers/attachments.css', true)
+    $xwiki.jsfx.use('uicomponents/widgets/liveData.js', true)
```

Comment:
Would be nice to load all the above JavaScript and CSS on demand, lazy, only when the Attachments tab is loaded, using something like https://github.com/xwiki/xwiki-platform/blob/master/xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-templates/src/main/resources/templates/display.vm#L49-L50 .